src/core/createInstance.js

```diff
@@ -121,7 +126,19 @@ export default (
     // underlying function call.
     // Also note that executeCommand may or may not return a promise.
     new Promise(_resolve => {
-      _resolve(executeCommand(commandName, options));
+      logger.log(`Executing ${commandName} command.`, "Options:", options);
+      _resolve(
+        // Note that promiseAllObject also clones the options object.
```

Comment:
I think it's sufficient to put it in the util's JSdoc, which I'll add another commit for. I think I feel that way because of other functions that do things similar to cloning where the cloning part isn't called out specifically in the name (e.g., `Array.prototype.concat`). I just want to make sure that we (I?) add a test for cloned options when we add tests for the core modules. I think cloning the options object is a good thing and something we should be doing regardless of `resolvePromisesAndClone`, but more importantly I don't want to accidentally go from cloning options to not cloning options later, since it could result in a negative change in behavior for the user.

Reply:
I've added a commit intended to improve the jsdoc in this regard.